src/main/kotlin/com/cjbooms/fabrikt/generators/controller/MicronautControllerInterfaceGenerator.kt

```diff
@@ -62,13 +56,14 @@ class MicronautControllerInterfaceGenerator(
         val methodName = methodName(op, verb, path.pathString.isSingleResource())
         val returnType = MicronautImports.RESPONSE.parameterizedBy(op.happyPathResponse(packages.base))
         val parameters = op.toIncomingParameters(packages.base, path.parameters, emptyList())
+        val globalSecurity = this.api.openApi3.getSecurityRequirements()
```

Comment:
**suggestion:** Would it make sense to call `securityOption` here already?

Reply:
done